package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "punchcard-content-types",
-  "description": "",
+  "description": "Combines with input-plugin to create Content Types in the Punchcard CMS",
```

Comment:
Great find! Can you please change the text to:
`Combines with input plugins to create forms with validation. Creates forms for Punchcard CMS.`
thank you @Haroenv